go/src/koding/socialapi/transport_test.go

```diff
@@ -0,0 +1,142 @@
+package socialapi_test
+
+import (
+	"encoding/json"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"testing"
+
+	"koding/kites/tunnelproxy/discover/discovertest"
+	"koding/socialapi"
+)
+
+func TestTransport(t *testing.T) {
+	var storage TrxStorage
```

Comment:
Done in https://github.com/koding/koding/commit/28fc46282c43fa1c87fbec1908bd19894d652cdc.